docs/adding-a-page.md

```diff
@@ -98,7 +98,7 @@ export default (
 
 ### Starting the server
 
-Now that the component is setup we can run `npm start dev:search` and navigate to
+Now that the component is setup we can run `yarn dev:search` and navigate to
```

Comment:
Yeah I would remove any references to search (and admin) since it won't work the app is called admin, and the app is very close to being deleted.